specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -1696,6 +1696,13 @@
             "type": "string",
             "description": "The filter to apply to the operation."
           },
+          {
```

Comment:
Note for me: the spec about this
https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/howto/azure-ad-graph-api-supported-queries-filters-and-paging-options#expand